README.md

```diff
@@ -1,4 +1,4 @@
-# copycat
+# ![copycat](https://user-images.githubusercontent.com/1731223/167850970-584e6953-6543-4085-af5a-f9d8b7ffe988.png)
```

Comment:
Can also keep around the `# copycat` and put the logo below if it'd be better. Or any other idea. Also, maybe we just don't want to put the logo in the readme at all? I'm happy either way.

Reply:
Looks awesome!